docs/installation/cloud-ksql-migration-guide.rst

```diff
@@ -0,0 +1,111 @@
+.. _cloud-ksql-migration-guide:
+
+Migrate |ccloud| KSQL from the preview version to the production-available version
+##################################################################################
+
+To deliver the benefits of new features and improvements as soon as they become
+available, new versions of |ccloud| KSQL are deployed frequently. Most of these
+deployments are invisible to you as a user, requiring no action on your part.
+
+The production-available version of |ccloud| KSQL is not backward-compatible 
+with the preview version, but after the release of the production-available
+version, we're committed to preserving backward compatibility.
+
+Backward incompatibility comes in various forms, including changes to
+serialization formats, data models, and the SQL syntax. To continue running your
+application from the preview version of KSQL, you may need to migrate your
+workload to an application that's backed by the production-available version.
+This document provides a simple process to guide you through this migration.
+
+.. note::
+
+    We will never automatically upgrade your KSQL application to use a
+    backward-incompatible release.
+
+Port the application’s schema
+*****************************
+
+You can run the following commands by using the |ccloud| web interface or the
+KSQL CLI. If you choose to use the KSQL CLI, you may find the SPOOL command

```

Comment:
Should we link to the docs for using the KSQL CLI with CCloud KSQL here? We've seen a bunch of questions on how to do that.

Reply:
Probably a good idea, here's the link we should target when referencing the KSQL CLI:

https://docs.confluent.io/current/quickstart/cloud-quickstart/ksql.html#accessing-a-ksql-application-in-ccloud-with-api-keys